client/src/modules/events/pages/eventPage.tsx

```diff
@@ -80,6 +85,11 @@ export const EventPage: NextPage = () => {
               }
             : { title: 'You canceled your RSVP 👋', status: 'error' },
         );
+        if (add) {
+          await initUserInterestForChapter({
+            variables: { event_id: 3 },
```

Comment:
```suggestion
            variables: { event_id: id },
```
Sorry, I missed this the first time!

Reply:
@ojeytonwilliams , Oops!, Sorry, I missed it too. I'll update and merge the changes.